R/expr-interp.R

```diff
@@ -95,34 +95,20 @@ NULL
 #' @export
 #' @rdname quasiquotation
 UQ <- function(x) {
-  x
+  abort("`UQ()` can't be called directly")
```

Comment:
Add something like "Check that you are in a quasiquoted argument" ?

Reply:
@hadley How about:

```
"`UQ()` can only be called as part of a quasiquoted argument"
```

Or:

```
"`UQ()` can only be called within a quasiquoted argument"
```

Reply:
The second I think. Maybe used instead of called?